experimental/cevobuild/Tests/CevoCoreTests/XCTestManifests.swift

```diff
@@ -0,0 +1,18 @@
+#if !canImport(ObjectiveC)
```

Comment:
You can kill this and use `swift test --enable-test-discovery`

Reply:
Sweet!